studio-docs/source/sharing-graphs.mdx

```diff
@@ -0,0 +1,498 @@
+import {EmbeddableExplorer} from 'gatsby-theme-apollo-docs';
+
+Apollo Studio provides features that help you share your graph with both teammates and external consumers.
+
+## The README page
+
+When a user opens one of your graph's variants in Apollo Studio, its **README page** is shown:
+
+Like the typical homepage for a Git repository, this page displays a Markdown-based README that you can edit with any content you like.
+
+> **Note:** Only organization members with the [`Org Admin`, `Graph Admin`, or `Contributor` role](./org/members/) can edit a variant's README. If a variant is [protected](./org/graphs/#protected-variants-enterprise-only), `Contributor`s _cannot_ edit its README.
+
+**Each variant has its own README.** If you edit one variant's README, those edits are _not_ applied to your graph's other variants. Among other uses, this enables you to provide different information to external consumers in a [public variant](#making-a-graph-public).
+
+### README shortcodes
+
+The README supports a collection of variables that you can include in its text. These variables are rendered with the correct value for the current variant.
+
+
+
+## Public graphs
+
+You can make individual [variants](./org/graphs/#managing-variants) of your graph public. If you do, **anyone with that variant's public link** can view the following pages for that variant in Studio:
+
+* Home
+* Schema
+* Explorer
+
+This enables consumers of your graph to learn about your graph's schema and any special usage information (such as authentication details) that you've documented in [the variant's README](#the-readme-page). They can also run properly authenticated operations against your graph with the Explorer.
+
+People outside your organization can't view any _other_ pages for a public variant (Fields, Operations, etc.), and they can never view _any_ pages for private variants. New variants always start as private.
+
+### Making a variant public
+
+> **Note:** Only organization members with the [`Org Admin` or `Graph Admin` role](./org/members/) can toggle a variant's visibility.
+
+1. Go to your graph's Settings page and open the Access tab.
+2. Under the Variants section, toggle the switch for the variant you want to make public.
+3. Confirm your selection.
+
+You can toggle the switch back to make the variant private again.
+
+## Embedding the Explorer
+
+If you have a [public variant](#public-graphs) of your graph, you can embed the [Apollo Studio Explorer](./explorer/) in a webpage that you can then provide to your graph's consumers. This enables those consumers to test out operations from your own website.
+
+For example, here's an embedded Explorer for an Apollo example graph:
+
+<EmbeddableExplorer
+  graphRef="Apollo-Fullstack-Demo-o3tsz8@current"
+  graphEndpoint="https://apollo-fullstack-tutorial.herokuapp.com/"
+  graphSubscriptionEndpoint="wss://apollo-fullstack-tutorial.herokuapp.com/graphql"
+  styles={{width: '100%', height:'450px'}}
```

Comment:
```suggestion
  styles={{width: '100%', height:'450px', border: '0px', borderRadius: '4px'}}
```

Iframes by default have an ugly grey `frameBorder` we could remove it on the react component or here